polls/tool/src/java/org/sakaiproject/poll/tool/producers/ResultsProducer.java

```diff
@@ -135,9 +135,11 @@ public void fillComponents(UIContainer tofill, ViewParameters viewparams,
 		
 		//get the number of votes
 		int voters = pollVoteManager.getDisctinctVotersForPoll(poll);
-		//Object[] args = new Object[] { Integer.valueOf(voters).toString()};
-		if (poll.getMaxOptions()>1)
-			UIOutput.make(tofill,"poll-size",messageLocator.getMessage("results_poll_size",Integer.valueOf(voters).toString()));
+		int totalVoters = externalLogic.getNumberUsersCanVote();
+		float percentVoters = (float) voters / (float) totalVoters;
+		String statsVoters = Integer.valueOf(voters).toString() + " / " +Integer.valueOf(totalVoters).toString()+" ("+String.format("%.02f", percentVoters * 100)+" %)";
```

Comment:
Please use StringFormat for the entire String and avoid converting to String, you can write integers directly with %d